tests/integration/models/device.spec.coffee

```diff
@@ -753,6 +753,201 @@ describe 'Device Model', ->
 				.then ({ is_accessible_by_support_until__date }) ->
 					m.chai.expect(is_accessible_by_support_until__date).to.be.null
 
+	describe 'given a multicontainer application with a single offline device', ->
+
+		beforeEach ->
+			resin.models.application.create('FooBar', 'raspberry-pi').then (application) =>
```

Comment:
Would it make sense to make it a facade (with fewer or no arguments) that is part of the `setup.coffee` ?

Reply:
I might do this later, but I'm not going to for now, since it depends a little on how the rest of the multicontainer code works out.